src/drt/src/frBaseTypes.h

```diff
@@ -287,6 +287,14 @@ enum class frAccessPointEnum
   NearbyGrid = 4  // nearby grid or 1/2 grid
 };
 
+enum class RipUpMode
+{
+  UNKNOWN = -1,
+  ripupDRC = 0,
+  ripupAll = 1,
+  ripupAroundDRC = 2
```

Comment:
If the value is already part of the `RipUpMode` enum, I think it's a bit redundant to include `ripup` in the value name. e.g.
```
  UNKNOWN = -1,
  DRC = 0,
  ALL = 1,
  AROUND_DRC = 2
```

I think the style guidelines also prefer enums to be all capital letters.
Then the value would simply be e.g. `RipUpMode::DRC`